osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -151,5 +172,53 @@ public void UpdateBeatmap(WorkingBeatmap beatmap)
                 Add(d);
             });
         }
+
+        private string getBPMRange(Beatmap beatmap)
+        {
+            double bpmMax = double.MinValue;
+            double bpmMin = double.MaxValue;
+            double bpmMost = beatmap.BPMAt(beatmap.ControlPoints.Where(c => c.BeatLength != 0).GroupBy(c => c.BeatLength).OrderByDescending(grp => grp.Count()).First().First().Time);
+            foreach (ControlPoint a in beatmap.ControlPoints)
+            {
+                if (a.BeatLength == 0) continue;
+                double tmp = beatmap.BPMAt(a.Time);
+                if (bpmMax < tmp) bpmMax = tmp;
+                if (bpmMin > tmp) bpmMin = tmp;
+            }
+            if (bpmMax == bpmMin) return Math.Round(bpmMin) + "bpm";
+            return Math.Round(bpmMin) + "-" + Math.Round(bpmMax) + "(" + Math.Round(bpmMost) + ")bpm";
+        }
+        
+        private Container infoLabel(FontAwesome icon, string text)
+        {
+            return new Container
+            {
+                AutoSizeAxes = Axes.Both,
+                Children = new[] 
+                {
+                    new TextAwesome
+                    {
+                        Icon = FontAwesome.fa_square,
+                        Colour = new Color4(68,17,136,255),

```

Comment:
missing spaces after commas. use an IDE which allows for applying standard code formatting.